src/Maps/MapData.h

```diff
@@ -24,6 +24,22 @@ enum class CellType;
 class MapData
 {
 public:
+	MapData();
+	MapData(const MapHeader& mapHeader);
+
+	MapHeader CreateHeader() const;
+
+	inline void SetVersionTag(int32_t versionTag) { MapData::versionTag = versionTag; };
```

Comment:
I'll switch it to use `this->`. I was trying to use `this.` with the dot operator instead of the arrow operator. The dot operator is used in C# syntax instead of the arrow and I was frustrated that I couldn't get it to work using `this`.